### PR TITLE
chore: working state for npm start and npm run build

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** fix webpack build for dev

JIRA: https://2u-internal.atlassian.net/browse/AU-1640

**What changed?**

- add overwrite webpack cli package from `@edx/frontend-build` for some bug
- Remove webpack because we are using webpack 5 from `@edx/frontend-build`
- remove dev build from `Make static` since it isn't being use. It is unnecessary for deployment.

**Testing Instructions**

- [ ] `rm -rf node_modules`
- [ ] `npm ci`
- [ ] `npm run build` (require restart lms & cms on devstack)
- [ ] `npm start`(require restart lms & cms on devstack)
  - [ ] this should be able to hot reload

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,5 +54,10 @@
     "test": "./node_modules/karma/bin/karma start --reporters spec,coverage",
     "lint": "fedx-scripts eslint --ext .jsx,.js openassessment/xblock/static/js/src/",
     "lint-fix": "fedx-scripts eslint --ext .jsx,.js openassessment/xblock/static/js/src/ --fix"
+  },
```
Comment (mattcarter) [inline package.json:57]: does this override belong here, in the `openedx` repo?

Comment (leangseu-edx) [reply package.json:57]: Because this fix apply to older version of frontend-build. I don't mind to just override it instead going through the whole deployment.

**Review by mattcarter — APPROVED** [review]
approved

**Review by jansenk — APPROVED** [review]
Squash conflits and this is good to go

**Comment by leangseu-edx** [comment]
I will not bump the version or push release because there isn't a code change. It only affects developer.
